web-packages/imgui-demo/src/panels/node-graph/visual-graph/base-graph.ts

```
@@ -0,0 +1,841 @@
+      node.m_ID = '';
+      this.m_nodes.splice(index, 1);
+      this.PostDestroyNode(nodeID);
+      this.EndModification();
+    }
+  }
+
+  //   abstract CanCreateNode (pNodeTypeInfo: TypeInfo): boolean;
+
+  isSubclassOf (derivedCtor: Function, baseCtor: Function): boolean {
+    // eslint-disable-next-line no-prototype-builtins
+    return derivedCtor === baseCtor || baseCtor.prototype.isPrototypeOf(derivedCtor.prototype);
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'isPrototypeOf' from target object.



(lint/suspicious/noPrototypeBuiltins)